lib/tasks/cleanup_needs_by_priority.rake

```diff
@@ -0,0 +1,6 @@
+namespace :cleanup_needs_by_priority do
+  desc 'Detele needs and notes records for needs.supplemental_data.food_priority == 3.'
+  task food_priority_3: :environment do
+    p Need.where('supplemental_data @> ?', { food_priority: '3' }.to_json).destroy_all
```

Comment:
Will this also work if the need has notes?

Reply:
yep tested locally, seems to work there

Reply:
yes, `destroy_all` takes care of the related data